truss/remote/baseten/core.py

```diff
@@ -120,6 +120,8 @@ def archive_truss(truss_handle: TrussHandle) -> IO:
     truss_ignore_file = truss_dir / ".truss_ignore"
     if truss_ignore_file.exists():
         ignore_patterns = load_trussignore_patterns(truss_ignore_file=truss_ignore_file)
+    else:
+        ignore_patterns = load_trussignore_patterns()
```

Comment:
I think this is from my change (https://github.com/basetenlabs/truss/commit/66c794b08553e25486f4a585250524833764bae8), not sure why it's showing up in this diff. Maybe try merging `main`?